### PR TITLE
Add DEX wallet settings page

### PR DESCRIPTION
**Pull request by song50119**
Work towards https://github.com/planetdecred/godcr/issues/619.
Related https://github.com/planetdecred/godcr/pull/722#issuecomment-1046795051
- Add Dex wallets setting page to create DCR, BTC a wallet

### CODE REVIEW

File: ui/page/dexclient/add_dex_modal.go [hunk 1]
```diff
@@ -21,34 +18,59 @@ import (
...
+	// without requesting password input on UI
+	defaultAppPass string
+	appPassword    decredmaterial.Editor
+	onDexAdded     func(*core.Exchange)
+
+	listServerClickable map[string]*decredmaterial.Clickable
+	selectedServer      string
+	listServerBtn       decredmaterial.Button
+	customServerBtn     decredmaterial.Button
+	isUseCustomServer   bool
```
Comment (itswisdomagain) [inline ui/page/dexclient/add_dex_modal.go]: These seem to be related to registering a DEX. Does this PR cover that feature as well? It'll be much easier to review if both features are submitted as separate PRs.

**Comment by dreacot** [comment]
Screenshot from 2022-03-14 16-46-47


1. Clicking on create bitcoin wallet, crashes with the following error;

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x12dd1ce]

goroutine 3087 [running]:
github.com/planetdecred/godcr/ui/page/components.(*AccountSelector).UpdateSelectedAccountBalance(0xc006f6ccb0)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/components/account_selector.go:122 +0x2e
github.com/planetdecred/godcr/ui/page/components.(*AccountSelector).ListenForTxNotifications.func1(0xc006f6ccb0, 0x1ebb618, 0xc000546b80)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/components/account_selector.go:216 +0x11c
created by github.com/planetdecred/godcr/ui/page/components.(*AccountSelector).ListenForTxNotifications
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/components/account_selector.go:203 +0x53
```

2. 
Screenshot from 2022-03-14 16-46-37

add a space before the wallet icon

3. why does the homepage show registration successful, shouldn't there be some more info on the dex home screen?
Screenshot from 2022-03-14 16-44-49
